LayoutTests/http/tests/iframe-monitor/eligibility.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/js-test-resources/js-test.js"></script>
+<script src="./resources/monitor-setup.js"></script>
+</head>
+<body>
+<script>
+
+description("Test resource monitor.");
+
+onload = async () => {
+    testRunner.waitUntilDone();
```

Comment:
use `jsTestIsAsync = true;` right after the `description()` call above, instead.